daft/internal/treenode.py

```diff
@@ -73,6 +71,8 @@ def to_dot_file(self, filename: str | None = None) -> str:
         return filename
 
     def to_dot(self) -> str:
+        import pydot
```

Comment:
lets put this in a try catch with the proper error message saying you need the dep